### ISSUE TITLE
REPL locks up when starting julia 1.11 with multiple threads

### ISSUE THREAD

**Issue by jwahlstrand** [JuliaGtk/Gtk4.jl#78]
Reported here https://github.com/jwahlstrand/AravisCameras.jl/issues/5 but it's a Gtk4 issue. I can reproduce it on Linux but have no time to look into it today.

It doesn't happen with Julia 1.10 and I haven't seen it on Windows.

**Comment by srikumarks** [comment]
Just encountered this. It looks like it is not locked entirely (at least on MacOS), every key stroke take several seconds to be delivered, but they eventually do get delivered. So this is perhaps due to some cross-event loop communication issues (between Julia's event loop and Gtk's) ... perhaps.

**Comment by jwahlstrand** [comment]
Thanks, good to know it's an issue on MacOS too. I do think it's an event loop conflict of some kind but am not sure what do to about it. For me at least, it can be avoided by starting Julia with `julia -t N,1` (or setting the environment variable `JULIA_NUM_THREADS = N,1`). It deserves a note in the README and the documentation.

**Comment by RechercheStochastique** [comment]
It happens on Windows 11 as well.

**Comment by jwahlstrand** [comment]
Wow, this issue seems to be gone in Julia 1.13. I know I ran into it in one of the release candidates, so the fix must have happened recently. I'll check on Windows and Mac before closing. It would be great if someone else could also confirm -- all you have to do is start julia with `julia -t1,2` and then `using Gtk4`. If you can type in the REPL after doing that then things are fixed.

**Comment by cmey** [comment]
Yes, on Windows 11 + Gtk4 v0.7.13 + Julia 1.13.0 started with  `julia --project=. -t 1,2`, `using Gtk4`, the REPL seems to be totally fine 👍
